docker/Dockerfile.deps

```diff
@@ -71,6 +71,14 @@ RUN LIBTIFF_VERSION=4.1.0 && \
     cd && \
     rm -rf /tmp/tiff-${LIBTIFF_VERSION}
 
+RUN OPENJPEG_VERSION=2.3.1 && \
```

Comment:
Please add to (you can use libjpeg-turbo recipe as a reference):
- Dockerfile.build.aarch64-linux
- Dockerfile.build.aarch64-qnx
- conda OpenCV (dali_opencv/recipe/meta.yaml)

Reply:
done